server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -450,6 +450,11 @@ private void upgradeZookeeper() {
         moveRootTabletToRootTable(zoo);
 
         // add system namespace permissions to existing users
+        // N.B. this section is ignoring the configured PermissionHandler
```

Comment:
Please avoid jargon/abbreviations such as "N.B."; it's more likely to cause confusion than to help reader understand the comment.